app/models/solid_errors/occurrence.rb

```diff
@@ -23,5 +23,13 @@ def parse_backtrace(backtrace)
     def send_email
       ErrorMailer.error_occurred(self).deliver_later
     end
+
+    def under_limit?
+      return true if error.occurrences.count == 1 || !SolidErrors.one_email_per_occurrence
+
+      error.occurrences.where("created_at > ?", error.prev_resolved_at).count == 1
+    rescue
```

Comment:
What errors have you experienced to warrant this `rescue`?

Reply:
Honestly, this is probably just an unnecessary rescue. Admittedly, I tend to over-rescue at times (imo)